### DISCUSSION TITLE
Logging into filebrowser server from another server

### DISCUSSION THREAD

**Discussion by q-omar** [filebrowser/filebrowser#1416] [Q&A]
Suppose I am running filebrowser on one server and I am trying to get users to log in. I click some click that gets  redirected to the filebrowser site. However instead of coming to the login page they get put directly into system based on who they are from original site. 

Now of course I would go ahead and actually create the users on the filebrowser site that correspond to users on original site. I looked into authentication methods and I assume the proxy one would suit me best?  Is there any more documentation on this?

I also played around with posting json to the /api/login endpoint and while I was able to store the response back as a jwt token on my original server inside localStorage, given the domains switch to the different domain local storage doesn’t persist.

**Comment by ramiresviana** [comment]
Using the proxy authentication method allows you to control the authentication. You can try creating a proxy server with different urls for the users like http://example.com/userA and http://example.com/userB. The documentation is available at https://filebrowser.org/configuration/authentication-method#proxy-header.

**Reply by q-omar** [reply]
I'm able to send get request and provide header like 'X-My-Header: admin'. Then receive a response like eyJhbGci...back. So can I then use this like for example:

www.domain1.com has some link to www.filebrowserdomain2.com/files?auth=eyJhbGci...

and it logs into second domain via the link on domain1?
